test/src/main/java/org/apache/accumulo/test/functional/FunctionalTestUtils.java

```diff
@@ -218,4 +221,34 @@ private static FateStatus getFateStatus(ClientContext context, AccumuloCluster c
       throw new RuntimeException(e);
     }
   }
+
+  /**
+   * Verify that flush ID gets updated properly and is the same for all tablets.
+   */
+  static long checkFlushId(ClientContext c, TableId tableId, long prevFlushID) throws Exception {
+    try (var metaScan = c.getAmple().readTablets().forTable(tableId).fetch(FLUSH_ID).build()) {
```

Comment:
Realized one more thing, I think this may only return tablets were the column is present.  If a tablet didn't have this column, then it may not even show up (like there would not even be tablet metadate entry of not present).  Because we are only fetching the flush_id column in the underly scan, tablet w/o that column do not show up.  Adding the check consistency will make it fetch the prev row col also which will make tablets w/o the column visible.  Also get the benefit of ensure the tablets are consistent.

```suggestion
    try (var metaScan = c.getAmple().readTablets().forTable(tableId).fetch(FLUSH_ID).checkConsistency().build()) {
```